GameLogic/engine/StandardFENSerializer.cs

```diff
@@ -4,8 +4,11 @@
 
 namespace GameLogic
 {
-    public static class SerializeHelper
+    public static class StandardFENSerializer
```

Comment:
Nice idea but why there is no interface base class for serializer?

Reply:
Static classes cannot have interfaces in C#

Reply:
I believe cardinal-evgeniy@'s question is "Why is a static class and not a interface + implementation"?

Reply:
I want to have a static class. I think it makes more sense.

Reply:
Fine for me also.